src/ui/EditTask.svelte

```diff
@@ -68,23 +68,45 @@
     const priorityOptions: {
             value: typeof editableTask.priority,
             label: string,
-            symbol: string }[] =
+            symbol: string,
+            accessKey: string,
+            accessKeyIndex: number}[] =
         [{
+            value: 'lowest',
+            label: 'Lowest',
+            symbol: prioritySymbols.Lowest,
+            accessKey: 'o',
+            accessKeyIndex: 1
+        }, {
             value: 'low',
             label: 'Low',
-            symbol: prioritySymbols.Low
+            symbol: prioritySymbols.Low,
+            accessKey: 'l',
+            accessKeyIndex: 0
         }, {
             value: 'none',
             label: 'Normal',
-            symbol: prioritySymbols.None
+            symbol: prioritySymbols.None,
+            accessKey: 'n',
+            accessKeyIndex: 0
         }, {
             value: 'medium',
             label: 'Medium',
-            symbol: prioritySymbols.Medium
+            symbol: prioritySymbols.Medium,
+            accessKey: 'm',
+            accessKeyIndex: 0
         }, {
             value: 'high',
             label: 'High',
-            symbol: prioritySymbols.High
+            symbol: prioritySymbols.High,
+            accessKey: 'h',
+            accessKeyIndex: 0
+        }, {
+            value: 'highest',
+            label: 'Highest',
+            symbol: prioritySymbols.Highest,
+            accessKey: 'e',
+            accessKeyIndex: 4
```

Comment:
Sorry but `e` has already been taken in an earlier in-progress PR for the Reminder field: #1925

I suggest using `i` for Highest instead.

Reply:
ok, changed to `i`